src/components/OneMonsterItem.js

```diff
@@ -1,56 +1,78 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import { NavLink } from 'react-router-dom'
 import styled from 'styled-components'
+import listlogo from './images/list.png'
 
 export default function OneMonsterItem({ monster }) {
   return (
-    <Link to={'/monster/' + monster.index}>
+    <ComponentContainer>
       <ContainerStyled>
         <MonsterNameStyled>{monster.name}</MonsterNameStyled>
         <ContainerTypeCR>
           <ParagraphType>{monster.type}</ParagraphType>
-          <ParagraphCR>{monster.challenge_rating}</ParagraphCR>
+          <ParagraphCR>CR: {monster.challenge_rating}</ParagraphCR>
         </ContainerTypeCR>
       </ContainerStyled>
-    </Link>
+      <NavLink to={'/monster/' + monster.index}>
+        <DetailLink src={listlogo} alt="view details" />
+      </NavLink>
+    </ComponentContainer>
   )
 }
+const ComponentContainer = styled.div`
+  display: flex;
+  padding: 2px;
+  align-items: center;
+`
+
 const ContainerStyled = styled.div`
-  width: 70%;
-  border-radius: 12px;
-  margin: 10px 15%;
+  width: 100%;
+  margin: 4px 0;
+  padding: 10px;
   display: inline-block;
   text-decoration: none;
+  color: var(--textblack);
+  border-bottom: 1px solid #d0d0cd;
 `
 const MonsterNameStyled = styled.h2`
-  font-size: 80%;
-  font-weight: 1000;
+  color: #262626;
+  font-size: 100%;
   text-align: left;
-  margin: 0;
+  margin: 0 0 7px 0;
   vertical-align: text-bottom;
-  border-bottom: 2px solid #f50808;
-  border-left: 2px solid #f50808;
+  font-family: serif;
   line-height: min-content;
-  text-decoration: none;
   padding-left: 2px;
 `
 const ContainerTypeCR = styled.div`
   display: flex;
 `
 const ParagraphType = styled.p`
   text-transform: capitalize;
-  font-size: 80%;
+  font-size: 15px;
   flex: 1 1;
   margin: 0;
+  padding-left: 2px;
   text-align: left;
   border-left: 2px solid #ffffff;
-  text-decoration: none;
-  padding-left: 2px;
 `
 const ParagraphCR = styled.p`
-  font-size: 80%;
+  font-size: 15px;
   flex: 1 1;
   margin: 0;
   text-align: right;
   text-decoration: none;
 `
+const DetailLink = styled.img`
+  max-height: 50px;
+  max-width: 50px;
+  background: linear-gradient(
+    155deg,
+    rgba(249, 247, 247, 1) 0%,
+    rgba(220, 20, 60, 1) 60%
+  );
+  border-radius: 6px;
+  padding: 10px;
+  margin: 5px 10px;
+  color: white;
+`
```

Comment:
Your naming of your styled components is a little inconsistent / confusing to me. Sometimes you add "Styled" at the end of your components, sometimes you don't - even if it is styled as well